pkg/utils/default_ocm_client.go

```diff
@@ -0,0 +1,146 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+const (
+	apiURLStage    = "https://api.stage.openshift.com"
+	ocmTokenEnvVar = "OCM_TOKEN"
+)
+
+// OCMResponseError is used to wrap HTTP error (400 - 599) response codes
+// which are returned from a request to OCM.
+type OCMResponseError int
+
+func (e OCMResponseError) Error() string {
+	return fmt.Sprintf("ocm responded with code %d", e)
+}
+
+func (e OCMResponseError) ServerSide() bool {
+	code := int(e)
+
+	return code >= 500 && code < 600
+}
+
+// NewDefaultOCMClient takes a variadic slice of options to configure a default
+// OCM client and applies defaults if no appropriate option is given. An error
+// may be returned if an unusable OCM token is provided or a connection cannot
+// be initialized otherwise the default client is returned.
+func NewDefaultOCMClient(opts ...DefaultOCMClientOption) (*DefaultOCMClient, error) {
+	var (
+		client DefaultOCMClient
+		err    error
+	)
+
+	for _, opt := range opts {
+		client.Option(opt)
+	}
+
+	if client.apiURL == "" {
+		client.apiURL = apiURLStage
+	}
+
+	if client.token == "" {
```

Comment:
I would not keep the `token` in memory. Would use a throw-away var, and not save it in the struct. Could cause memory leaks. So basically remove L#49-54 here.

Reply:
I'll address the token related question here:

Yep can totally nuke this from the struct. We could play around with refresh tokens if we need to, but it's not needed now.

> We will ever get OCM tokens from outside the env?

No clue, but I can remove the option and add some way to make it easy to inject the dependency for mocking.